var/spack/repos/builtin/packages/py-gevent/package.py

```diff
@@ -35,3 +35,7 @@ class PyGevent(PythonPackage):
     depends_on('py-greenlet@0.4.13:', when='@:1.4', type=('build', 'run'))
     depends_on('py-zope-event', when='@20.5.1:', type=('build', 'run'))
     depends_on('py-zope-interface', when='@20.5.1:', type=('build', 'run'))
+
+    patch('icc.patch',
+          sha256='44a3aa7ec3a9f056145c25492012f494166dee4c816a3a998a7e3b2d85b57c77',
```

Comment:
No need for a sha256 if it's a local file. Also, does this patch apply to all versions of the package?

Reply:
I double checked, it works with all versions.

I had to add the sha to make it work at first. It works without sha now on a fresh Spack installation.